FAQ.md

```diff
@@ -17,7 +17,7 @@ Frequently Asked Questions
 In 3.x release time OKD was used as an upstream project for Openshift Container Platform. OKD could be installed on 
 Fedora/CentOS/RHEL and used CentOS based images to install the cluster. OCP, however, could be installed only on RHEL and its images were rebuilt to be RHEL-based.
 
-[Universal Base Image project](https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image) has enabled us to run RHEL-based images on any platform, so the full image rebuild is no longer necessary, allowing OKD4 project to reuse most images from OCP4. There is another critical part of OCP - Red Hat Enterprise Linux CoreOS. Although it's an open source project (since it's essentially RHEL8) it's not a community-driven project. As a result OKD workgroup has 
+[Universal Base Image project](https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image) has enabled us to run RHEL-based images on any platform, so the full image rebuild is no longer necessary, allowing OKD4 project to reuse most images from OCP4. There is another critical part of OCP - Red Hat Enterprise Linux CoreOS. Although RHCOS is an open source project (much like RHEL8) it's not a community-driven project. As a result, OKD workgroup has 
```

Comment:
"much like" is a bit misleading. RHCOS == RHEL8 content.

Lets sort this out later though